File: servicces/src/doan.ProjectManagement.Application.Contracts/ProjectTasks/Dto/GetProjectTaskForInputDto.cs

using System;
using Volo.Abp.Application.Dtos;

namespace doan.ProjectManagement.ProjectTasks.Dto
{
    public class GetProjectTaskForInputDto : PagedAndSortedResultRequestDto
    {
        public string Filter { get; set; }
        public Guid ProjectInformationId { get; set; }
    }
}

File: applications/admin-web/src/app/shared/services/project/project-task/models/get-project-task-for-input-dto.ts
import { PagedAndSortedResultRequestDto } from '@abp/ng.core';

export class GetProjectTaskForInputDto extends PagedAndSortedResultRequestDto {
  filter: string;
  projectInformationId: string;

  constructor(initial: Partial<GetProjectTaskForInputDto> = {}) {
    super(initial);
  }
}
